.gitignore

```diff
@@ -158,3 +158,8 @@ sdk/cosmos/azure-cosmos/test/test_config.py
 
 # temporary folder to refresh SDK with cadl
 TempTypeSpecFiles/
+
+# Eclipse project files
```

Comment:
Why are we touching cosmos files?

Reply:
@johanste not sure who edited this file, but I see now that it's not part of this PR, so this change was likely accidental and was already reverted. Thanks!